src/client/src/components/Item/Item.jsx

```diff
@@ -9,7 +9,7 @@ import "./item.css";
 export const Item = ({ item, handleItemDelete, HandleItemUpdate }) => {
```

Comment:
Add Component postfix to both all class/function names and file names (ex: item-component.jsx)